azure-pipelines.yml

```
@@ -0,0 +1,33 @@
+# Node.js
+# Build a general Node.js project with npm.
+# Add steps that analyze code, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/javascript
+
+
+pool:
+  vmImage: ubuntu-latest
+
+steps:
+- task: NodeTool@0
+  inputs:
+    versionSpec: '16.x'
+  displayName: 'Install Node.js'
+
+- script: |
+    npm install
```

```suggestion
    npm ci
```

https://dev-tips.com/nodejs/difference-between-npm-install-and-npm-ci

---

TIL about npm ci TY @baywet